Makefile

```diff
@@ -46,7 +46,13 @@ build-e2e:
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
```

Comment:
Would
```
define test =
	go test -timeout 90m \
		-v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
		-args -v 5 -logtostderr \
		$1 $2 $3 $4 $5 $6 $7 $8 $9
endef

.PHONY: test-e2e
test-e2e: ## Run openshift specific e2e test
	$(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.focus "Feature:Operators")
	$(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.skip "Feature:Operators")
```
serve the same purpose?

Reply:
What's the second case supposed to do if the first focuses on the same thing? Is this a typo?

Reply:
https://jira.coreos.com/browse/CLOUD-458

Reply:
The first has `gingko.focus` which just runs anything that complies to `Feature:Operators` regex. The second one has `gingko.skip` which runs anything but specs that complies to `Feature:Operators` regex.

Reply:
@michaelgugino can you also add a comment into the Makefile explaining why this is needed? E.g. `//The primary reason for this is to preserve logs for troubleshooting test failures and flakes in the tests. Feature:Operator tests remove deployments. Thus loosing all the logs previously acquired.`

Reply:
@ingvagabund done.